### ISSUE TITLE
Change the switch of runs rule

### ISSUE THREAD

**Issue by leanderb** [robocup-at-work/rulebook#47]
from Discord rule-book-discussion-2022 channel:
1. Change the switch of runs rule:
Replacing a lower scoring run should be allowed bc eg BMT is higher than PPT and RTT which most teams will discard first if they face onsite issues.
Points could ceil at the replaced runs score, or you could make some kind of percentage (tbd)

**Comment by marcomasa** [comment]
Maybe we could also create some sort of table/graph to visualize the potential points and difficulty for each test.
This would help teams to understand the required tasks and therefore prioritize work packages. Some might want to focus on BMT / BTTs rather than PPT or RTT.
They should however still not "usually" perform in other slots than the regular schedule.

**Comment by steup** [comment]
I suggest we limit the replacement to a fraction of all runs, to prevent excessive replacements and go with a percentage-based awarding of points (compute how much of the run was successful and award this percentage of the points of the run, that was replaced.) The remaining issues will be regarding the time. How to handle replacement tasks, which are longer than the replaced task.

**Comment by leanderb** [comment]
- it is only allowed to move a task to another time slot which is scheduled later (the run time of the tests increases from slot to slot, so the schedule remains)
- also the maximum points increases from task to task, so a penalty / point reduction is not necessary anymore
TODO TC: check rulebook

**Comment by marcomasa** [comment]
I removed the sentence regarding the score, also rephrased it a little bit so it is more clear.

I had to add one thing:
You cannot repeat a test that has been replaced.

why?:
If a team replaces BTT1 with BMT, then BTT2 with BTT1 etc., they are creating their own custom schedule.
I would like teams to all stick to the original plan if possible.

how should they do?:
If they struggle with BTT1, they can do this one (much) later during ATT1 etc. 
We have the teams performing for "longer" like that and not too much swapping around.

Closed with [d599dd6dd28a5446ba9512d9961d0fbe207eaf8b](https://github.com/robocup-at-work/rulebook/commit/d599dd6dd28a5446ba9512d9961d0fbe207eaf8b)
